spec.bs

```diff
@@ -438,6 +452,8 @@ A [=component=] has an associated <dfn for=component>regular expression</dfn>, a
 
 A [=component=] has an associated <dfn for=component>group name list</dfn>, a [=list=] of strings, which must be set upon creation.
 
+A [=component=] has an associated <dfn for=component>has regexp groups flag</dfn>, a [=boolean=], which must be set upon creation.
```

Comment:
These days we're trying to move away from "flags" which can be off or on, toward just booleans which are true or false. Having something that's a boolean with flag in its name is a bit confusing. So I'd just say "has regexp groups".